config/locales/en.yml

```diff
@@ -145,7 +145,7 @@ en:
    available_groups: 'Available Groups'
    joined_groups: 'Joined Groups'
    leader: '(Leader)'
-   instructions: "You aren't a part of any groups yet! Click %{icon} to create one or join any available groups below.<br><br>Support groups are great for having open discussions about mental health! You can create groups, invite members, and organize meetup events online or offline. You can also join groups your allies have created."
+   instructions: "You aren't a part of any groups yet, but you can click %{icon} to create one!"
```

Comment:
Wouldn't this translation need to change for other languages too?

Reply:
yes, I wanted your approval before getting translation for 4 other languages ;) I noted that in the comments when I submitted the PR. ;)

Reply:
Thanks Andy!